submissions/Roka20012/js-dom/interactive.js

```diff
@@ -92,14 +89,14 @@ function checkStory({ target }) {
     if (target.tagName == "A") {
         for (let i = 0; i < nav.children.length; i++) {
             if (nav.children[i] == target) {
-            	if(!text.children[i].children.length) text.children[i].innerHTML = stories[i + 1];
-                delete stories[i+1];
+                if (!text.children[i].children.length) text.children[i].innerHTML = stories[i + 1];
+                delete stories[i + 1];
                 showText(i);
```

Comment:
do I understand right that here you move your stories[i] to text variable, and when all links in nav was visited your stories are empty and you use only text variable?

Reply:
That`s right

Reply:
This is not very good approach. First of all you mutate your input data. Second. You copy your data when it is unnecessarily. Actually you should copy data for avoiding mutating sometimes :)  Third. This is discussable, but the code isn't very clear to understand(to be hones for me very unclear). Maybe we can try to rewrite checkStory and showStory in another way? Just add classes to your navbars links, then add the same names to your data object. and then only change text in your text element. Feel free to ask questions here or in the chat room, if you don't understand something or don't agree with something :)

Reply:
> @Oodmincheg I add comments for clarity.
If it's still not clear I can rewrite it.

Point about clarification was 3d :) There are another 2 reasons why your option isn't nice. 
>I chose this option because the user can click on only 2 menu items and close the site. In my method, every story will only load once and then we just show it to the user and others stories will hide.

If I got you right, you try economy of memory if user click only 1 or 2 items and then close the site. But you already download all your data. so maby this economy unnecessary?

Reply:
I have transferred all my stories to html div.
Can i do that?or it is`nt nice too?

Reply:
@Oodmincheg 
In my opinion, this is not very good because if we have many elements we will need to specify a class or id for each. In my case, I just reassign one id to different elements. I'll think about it again, maybe I'm mistaken thanks.